### PR TITLE
kantamruy, kdam thmor: name tables fixed, version incremented

### LINKED ISSUE #1882: Change Designer text for 'Kantumruy' and 'Kdam Thmor'
Hello. Could you help change the Designer text of my two typefaces 'Kantumruy' and 'Kdam Thmor' description to my foundry?

Url: https://anagata.design
Twitter: https://twitter.com/sovichet

Designer text:

Sovichet is a Cambodian self-taught typeface designer based in Phnom Penh, Cambodia. He runs his studio [Anagata](https://anagata.design) offering brand identity and custom font design services. He also talks and teaches Khmer typeface design inside and outside Cambodia. He is interested in Old Khmer and other Southeast Asian scripts.

### PR DESCRIPTION
**Pull request by laerm0**
closes #1882

### CODE REVIEW

File: ofl/kdamthmor/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1,7 +1,2 @@
-<p>Kdam Thmor is a display face based on the writing style of a brush used on a wall. 
-It is designed for Khmer Unicode text. Kdam Thmor has an edgy style, a medium 
-size and is suitable for headings and large typography. It was designed in 
-2013 by <a href="http://it4ug.net/">Tep Sovichet</a>, a high school student 
-from Cambodia, who learned typeface design with local expert 
-<a href="http://khmertype.blogspot.com/">Danh Hong</a>. It is his second
-font published in Google Fonts.</p>
\ No newline at end of file
+<p>Kdam Thmor is a display face based on the writing style of a brush used on a wall. It is designed for Khmer Unicode text. Kdam Thmor has an edgy style, a medium size and is suitable for headings and large typography.</p>p>
```
Comment (m4rc1e) [inline ofl/kdamthmor/DESCRIPTION.en_us.html]: Fix the tag, `</p>p>`

**Comment by laerm0** [comment]
D'oh. Fixed that, sorry.

**Comment by RosaWagner** [comment]
This PR looks good to go, although now that I discovered the designer section in catalog/designer, I would be tempted to transfer the author informations there.

**Comment by RosaWagner** [comment]
Made a new PR to implement modifications: https://github.com/google/fonts/pull/2680

**Pull request closed without merging**